apps/designer/app/designer/features/style-panel/style-source-input/style-source.tsx

```diff
@@ -39,7 +41,75 @@ const menuCssVars = ({ show }: { show: boolean }) => ({
   [menuTriggerVisibilityVar]: show ? "visible" : "hidden",
 });
 
+const MenuTrigger = styled("button", baseButtonStyle, {
+  position: "absolute",
+  right: 0,
+  top: 0,
+  height: "100%",
+  padding: 0,
+  borderTopLeftRadius: 0,
+  borderBottomLeftRadius: 0,
+  visibility: cssVars.use(menuTriggerVisibilityVar),
+  variants: {
+    source: {
+      local: {
+        background: "$colors$backgroundStyleSourceToken",
```

Comment:
I think you don't need this background. Nothing changes when I toggle it on and off in browser

<img width="386" alt="Screenshot 2023-01-14 at 20 07 20" src="https://user-images.githubusercontent.com/825702/212482202-472c218f-8315-4178-9816-de3f546f29e1.png">

Same for other values of `source`

Reply:
yep, its the same color as the button, removed